integration_tests/library/libraries_test.go

```diff
@@ -16,9 +16,15 @@ const (
 	librariesE2ETest2 = "LIB-E2E-002"
 	librariesE2ETest3 = "LIB-E2E-003"
 	librariesE2ETest4 = "LIB-E2E-004"
+	librariesE2ETest5 = "LIB-E2E-005"
+	librariesE2ETest6 = "LIB-E2E-006"
+	librariesE2ETest7 = "LIB-E2E-007"
 )
 
 func TestLibrary(t *testing.T) {
+	renv := os.Getenv("PKGR_TESTS_SYS_RENV")
+	expectFail := renv == ""
```

Comment:
This variable doesn't apply to every test and therefore is a bit confusing. Again, if you need branching logic like this, I recommend separating the renv tests from the other kinds of tests.